pkg/codegen/nodejs/gen.go

```diff
@@ -2763,13 +2598,151 @@ func GeneratePackage(tool string, pkg *schema.Package,
 		}
 	}
 
+	if err = genBackwardsCompatibleTypesFiles(files, modules); err != nil {
```

Comment:
If we are interested in removing this call after a major version bump, we need to add a mechanism to disable this call somehow.